src/sentry/static/sentry/app/views/performance/vitalsCards.tsx

```diff
@@ -0,0 +1,167 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import {Location} from 'history';
+
+import Card from 'app/components/card';
+import Link from 'app/components/links/link';
+import {t} from 'app/locale';
+import space from 'app/styles/space';
+import {Organization} from 'app/types';
+import EventView from 'app/utils/discover/eventView';
+import {getAggregateAlias, WebVital} from 'app/utils/discover/fields';
+import {formatPercentage} from 'app/utils/formatters';
+import VitalsCardsDiscoverQuery from 'app/views/performance/vitalDetail/vitalsCardsDiscoverQuery';
+
+import {
+  vitalAbbreviations,
+  vitalDetailRouteWithQuery,
+  vitalsBaseFields,
+  vitalsThresholdFields,
+} from './vitalDetail/utils';
+
+type Props = {
+  eventView: EventView;
+  organization: Organization;
+  location: Location;
+};
+
+export default function VitalsCards(props: Props) {
+  const {eventView, organization, location} = props;
+  const vitalsView = eventView.clone();
+
+  const shownVitals = [
+    WebVital.FP,
+    WebVital.FCP,
+    WebVital.LCP,
+    WebVital.FID,
+    WebVital.CLS,
+  ];
+
+  return (
+    <VitalsCardsDiscoverQuery
+      eventView={vitalsView}
+      orgSlug={organization.slug}
+      location={location}
+    >
+      {({isLoading, tableData}) => (
+        <VitalsContainer>
```

Comment:
The CSS here might need some work, on narrower view ports, the cards go off the screen.

<img width="849" alt="Screen Shot 2020-11-27 at 4 38 11 PM" src="https://user-images.githubusercontent.com/10239353/100485671-f662d300-30ce-11eb-9a69-e40dc4b48577.png">

Reply:
I'm not sure about controlling for the widows, but we can fix it going off screen. 👍